app/helpers/exporters/attendee_csv.py

```diff
@@ -0,0 +1,23 @@
+from app.helpers.ticketing import TicketingManager
+
+class AttendeeCsv:
+	def __init__(self):
+		pass
+
+	@staticmethod
+	def export(event_id):
+		(_, event_id, holders, _, _) = TicketingManager.get_attendee_export_info(event_id=event_id)
+		headers = 'Order#,Order Date, Status, First Name, Last Name, Email, Country,' \
+				'Payment Type, Ticket Name, Ticket Price, Ticket Type'
+
+		fields = ('order_invoice', 'created_at', 'status', 'firstname', 'lastname', 'email', \
+				  'country', 'paid_via', 'ticket_name', 'ticket_price', 'ticket_type')
```

Comment:
This is one of the reasons why PEP8 recommends "space" over "tab". I personally prefer "tab", but...